src/server/utils/wallets/get-gcp-kms-account.ts

```diff
@@ -10,20 +10,19 @@ import {
 import { serializeTransaction } from "thirdweb/transaction";
 import { hashMessage } from "thirdweb/utils";
 import type { Account } from "thirdweb/wallets";
-import type {
-  SignableMessage,
-  TransactionSerializable,
-  TypedData,
-  TypedDataDefinition,
-} from "viem";
+import type { SignableMessage, TypedData, TypedDataDefinition } from "viem";
 import { hashTypedData } from "viem";
 import { getChain } from "../../../shared/utils/chain"; // Adjust import path as needed
 
 type SendTransactionResult = {
   transactionHash: Hex;
 };
 
-type SendTransactionOption = TransactionSerializable & {
+type SerializableTransaction = Awaited<
```

Comment:
Don't we already have this type exported in the SDK?

Reply:
Not exported, hance the struggle 😅

Reply:
Just export it! Put up a PR we can get it out today